src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/components/download-image/download-image.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, OnInit, Input, OnDestroy } from '@angular/core';
+import {
+  ContentService,
+  ContentDetail, ContentResolveBehavior, ContentDownloadLinkCreateRequest, Output
+} from '@picturepark/sdk-v1-angular';
+import { map } from 'rxjs/operators';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'pp-download-image',
+  templateUrl: './download-image.component.html',
+  styleUrls: ['./download-image.component.scss']
+})
+export class DownloadImageComponent implements OnInit, OnDestroy {
```

Comment:
@awaitDzn Rename to OutputDownloadMenuComponent

Reply:
@neotek done